AppCreationScripts/apps.json

```diff
@@ -0,0 +1,57 @@
+{
+    /*
```

Comment:
Perhaps Microsoft portal will remove this automatically, but ideally we should not add comments like this into a json file. JSON standard does not define/allow comment.